workflow-dev/tpl/generate_params.toml

```diff
@@ -158,24 +166,32 @@ secretPrefix = "private-registry"
 # Time in minutes after which the token should be refreshed.
 # Leave it empty to use the default provider time.
 tokenRefreshTime = ""
+# cpu = "100m"
+# memory = "50Mi"
 
 [registry_proxy]
 org = "deisci"
 pullPolicy = "Always"
 dockerTag = "canary"
 hostPort = 5555
+cpu = "100m"
+memory = "50Mi"
```

Comment:
Nope we have that set in the original chart https://github.com/deis/charts/blob/master/workflow-dev/tpl/deis-registry-proxy-daemon.yaml#L26-L29. Now I moved it to generate_param.toml